ultravox/tools/ds_tool.py

```diff
@@ -41,16 +54,36 @@ class TextGenerationTask:
     max_tokens: int = 128
     temperature: float = 0
 
+    def __post_init__(self):
+        if self.template.startswith("@"):
+            self.template = open(self.template[1:], "r").read()
```

Comment:
nit: suggest using `with`. The `open(...).read()` pattern won't close the file.
```suggestion
            with open(self.template[1:], "r") as template_file:
                self.template = template_file.read()
```